### PR TITLE
Moving query validations from Logger to query_db

### PR DESCRIPTION
**Pull request by oferKatz07**
Removed query method and validations from the logger to query_db.py which resides under the enterprise repo.
Additionally separated the ZMQ receive and send timeout values, since some processing is done before the response is returned

### CODE REVIEW

File: movai_core_shared/logger.py [hunk 1]
```diff
@@ -131,103 +131,6 @@ def get_logger(logger_name: str):
         logger.propagate = False
         return logger
 
-    @staticmethod
-    def get_logs(limit=1000, offset=0, level=None, tags=None, message=None, from_=None, to_=None, pagination=False,
-                 services=None):
```
Comment (engdorm) [inline movai_core_shared/logger.py:136]: don't delete this funcion
make it backwards compatible 
and keep the function and direct it to the new logic

File: movai_core_shared/envvars.py [hunk 2]
```diff
@@ -48,7 +48,8 @@
 ROS2_DISTRO = "dashing"
 # ZMQ environment variables
 MSG_HANDLER_LOCAL_CONN = os.getenv("MSG_HANDLER_LOCAL_CONN", "ipc:///run/movai/msg_handler_local_comm.ipc")
-MOVAI_ZMQ_TIMEOUT_MS = os.getenv("MOVAI_ZMQ_TIMEOUT_MS", 1000)
+MOVAI_ZMQ_RECV_TIMEOUT_MS = os.getenv("MOVAI_ZMQ_RECV_TIMEOUT_MS", 5000)
+MOVAI_ZMQ_SEND_TIMEOUT_MS = os.getenv("MOVAI_ZMQ_SEND_TIMEOUT_MS", 1000)
```
Comment (engdorm) [inline movai_core_shared/envvars.py:52]: I think we can set send without a timeout

File: movai_core_shared/envvars.py [hunk 3]
```diff
@@ -48,7 +48,8 @@
 ROS2_DISTRO = "dashing"
 # ZMQ environment variables
 MSG_HANDLER_LOCAL_CONN = os.getenv("MSG_HANDLER_LOCAL_CONN", "ipc:///run/movai/msg_handler_local_comm.ipc")
-MOVAI_ZMQ_TIMEOUT_MS = os.getenv("MOVAI_ZMQ_TIMEOUT_MS", 1000)
+MOVAI_ZMQ_RECV_TIMEOUT_MS = os.getenv("MOVAI_ZMQ_RECV_TIMEOUT_MS", 5000)
```
Comment (engdorm) [inline movai_core_shared/envvars.py:51]: maybe 5 seconds is too long 
the HTML request from the F.E will be set to NO RESPONSE by 5 seconds

**Review by engdorm — CHANGES_REQUESTED** [review]

File: movai_core_shared/envvars.py (hunk 2 above)
Comment (oferKatz07) [reply movai_core_shared/envvars.py:52]: You are correct

**Review by engdorm — APPROVED** [review]

File: movai_core_shared/envvars.py (hunk 3 above)
Comment (Mograbi) [reply movai_core_shared/envvars.py:51]: MS here is misleading between milliseconds and microseconds

**Comment by engdorm** [comment]
merged from another branch

**Pull request closed without merging**